doc/md/examples/grammar.txt

```diff
@@ -14,7 +14,7 @@
 
 <obj_sort> ::= 
     'object'
-    'persistent'? 'actor'
+    'persistent'? 'actor' ('[' <exp> ']')?
```

Comment:
I like the syntax although it might be confused with array syntax. An alternative could be `('migrate' <exp>)?` or, not sure, if we want to keep existing reserved keywords maybe`('with' <exp>)?`.

Reply:
It can be that simple:
``` Motoko
import { migration } = "Migration";

persistent actor (with migration) Frogger { ...
```

Reply:
That's cute, I guess. Luc shall I try to reuse the parenthetical syntax for this too, as gabor is suggesting. It avoids introducing a new keyword since migration is just a field name (but the explanation will be more convoluted - the argument has to be record with field migration of type etc....)

Reply:
Type checking is basically `par <: (Opt (Obj (Object, [{lab : "migration"; typ : Obj dom -> Obj cod }]))`, but you have to infer it first.